benchmark/http-benchmarkers.js

```diff
@@ -0,0 +1,168 @@
+'use strict';
+
+const child_process = require('child_process');
+
+function AutocannonBenchmarker() {
+  this.name = 'autocannon';
+
+  const autocannon_exe = process.platform === 'win32'
+                         ? 'autocannon.cmd'
+                         : 'autocannon';
+  this.present = function() {
```

Comment:
you should move these to the prototype, that is how all the other classes works.